indico/modules/events/templates/display/conference.html

```diff
@@ -49,6 +45,10 @@
                             {% if event.address %}
                                 <div class="address nohtml">{{ event.address }}</div>
                             {% endif %}
+
+                            {% if event.map_url or (event.room and event.room.map_url) %}
+                                <a href="{{ event.map_url or event.room.map_url }}" target="_blank">Go to map</a>
```

Comment:
I'm not a friend of `target="_blank"` - but *if* it's used, it should go along with `rel="noopener noreferrer"`

Reply:
From a UX perspective, `target="_blank"` is preferred. The user is navigating Indico and is expecting to remain in it when clicking links. I understand the need for `noopener`, but why `noreferrer`?

Reply:
Better privacy by not sending a referer to the target URL.

https://developers.google.com/web/tools/lighthouse/audits/noopener